src/components/breadcrumbs/examples/breadcrumbs-buttons.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, h, State } from '@stencil/core';
+import {
+    BreadcrumbItem,
+    LimelBreadcrumbsCustomEvent,
+} from '@limetech/lime-elements';
+
+/**
+ * Items as buttons
+ * The Breadcrumbs can also be used to navigate between different
+ * steps of a process, such as steps of a form or survey, or
+ * moving through steps of a wizard.
+ *
+ * In this case, you will not provide any `link`s and instead will
+ * handle the clicks. When no links are provided, the component
+ * will automatically generate a list of `buttons`s.
+ *
+ * Keep in mind that the last item will not be rendered as an
+ * HTML button and and therefore won't be clickable.
+ */
+@Component({
+    tag: 'limel-example-breadcrumbs-buttons',
+    shadow: true,
+})
+export class BreadcrumbsExampleButtons {
+    @State()
+    private items: BreadcrumbItem[] = [
+        {
+            text: 'Account info',
+        },
+        {
+            text: 'Personal info',
+        },
+        {
+            text: 'Payment info',
+        },
+        {
+            text: 'Confirm your details',
+        },
+    ];
+
+    @State()
+    private selectedItem: BreadcrumbItem;
+
+    public render() {
+        return [
+            <limel-breadcrumbs
+                items={this.items}
+                onSelect={this.handleSelect}
+            />,
+            <limel-example-value
+                label={'last select item'}
+                value={this.selectedItem?.text}
```

Comment:
```suggestion
                value={this.selectedItem.text}
```

Reply:
this change generates an error @Befkadu1 
![image](https://github.com/Lundalogik/lime-elements/assets/35954987/8ea3d0a8-251a-4796-bc02-1dec6f25ca40)

Reply:
Shouldn't we have a selectedItem from the beginning ? Here it is undefined at the beginning

Reply:
🤔 there is no selected item, or? 
The last step is always the "current" step, but it is never "selected". It's just an static thing. Or perhaps I misunderstand what you mean